ui_framework/components/form/_color_picker.scss

```diff
@@ -0,0 +1,29 @@
+.kuiColorSwatch {
+  width: 19px;
+  height: 19px;
+  border-radius: 3px;
+  border: 1px solid black;
```

Comment:
This is a great idea. If we use a box-shadow instead then we can even get the border to blend with the swatch color:

```scss
box-shadow: inset 0 0 0 1px rgba(#000, 0.2);
```

![image](https://user-images.githubusercontent.com/1238659/26956411-422ff078-4c72-11e7-97d9-f69c2bf68071.png)